main/src/cgeo/geocaching/connector/trackable/AbstractTrackableLoggingManager.java

```diff
@@ -26,5 +26,8 @@ public AbstractTrackableLoggingManager(final Context context) {
     public abstract boolean isRegistered();
 
     @Override
+    public abstract boolean isTrackingCodeNeededToPostNote();
```

Comment:
Like I wrote in the other PR, an abstract class with an abstract method overriding an interface method is a bit confusing. I would either leave the abstract method away or (even better) provide a useful default implementation to avoid overriding this is every single concrete subclass.